app/SharedExpense.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SharedExpense extends Model
```

Comment:
We can use a JSON array inside the expense to track all the id's of the users involved in the expense.